src/theory/rewriter.cpp

```diff
@@ -205,11 +205,9 @@ Node Rewriter::rewriteTo(theory::TheoryId theoryId,
   }
   // Rewrite until the stack is empty
   for (;;){
-
-    if (hasSmtEngine &&
-		d_iterationCount % ResourceManager::getFrequencyCount() == 0) {
+    if (hasSmtEngine)
```

Comment:
I think that we are able to get rid of `d_iterationCount` altogether.

Reply:
I think I've done that... `grep -Inri d_iterationCount src/` doesn't find anything...